backends/nxp/tests/test_quantizer.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2024 NXP
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Tests for NeutronQuantizer.
+
+import torch
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
+from torch.fx import GraphModule
+
+from executorch.backends.nxp.quantizer.neutron_quantizer import NeutronQuantizer
+import executorch.backends.nxp.tests.models as models
+
+
+def _get_target_name(node):
+    return node._pretty_print_target(node.target)
+
+
+def test_quantizer_conv2d():
+    model = models.Conv2dModule()
+    model.eval()
+
+    example_input = (torch.ones(1, 4, 32, 32),)
+    quantizer = NeutronQuantizer()
+    graph_module = torch.export.export_for_training(model, example_input, strict=True).module()
+
+    # noinspection PyTypeChecker
+    m = prepare_pt2e(graph_module, quantizer)
+    m(*example_input)
+    m = convert_pt2e(m)
+    # print(f"Quantized model: {m}")
+    nodes = list(m.graph.nodes)
+
+    assert len(nodes) == 11
+    assert nodes[7].name == "conv2d"
+    # [0]: Input, [1] : weights, [2]: bias
+    assert _get_target_name(nodes[7].args[0]) == 'torch.ops.quantized_decomposed.dequantize_per_tensor.default'
+    assert _get_target_name(nodes[7].args[1]) == 'torch.ops.quantized_decomposed.dequantize_per_tensor.default'
+    assert _get_target_name(nodes[7].args[2]) == 'torch.ops.quantized_decomposed.dequantize_per_tensor.default'
+    assert _get_target_name(nodes[8]) == 'torch.ops.quantized_decomposed.quantize_per_tensor.default'
+    assert nodes[8].args[0].name == "conv2d"
```

Comment:
can we also run it? Just to make sure the prepare/convert worked ok when consuming annotations, and the graph is still "functional"

Reply:
Thanks for the tip. Do you consider running `m(*example_input)` as enough in this case? As I have mentioned in previous comment, we plan to introduce some kind of `Tester` but we don't have this ready yet.